tests/integration/test_ols.py

```diff
@@ -29,7 +29,7 @@ def test_root() -> None:
 
 
 def test_status() -> None:
-    response = client.get("/")
+    response = client.get("/status")
```

Comment:
See previous comment about healthz

Reply:
dtto, there's function `test_root` for it